examples/app/logs.rs

```diff
@@ -8,6 +8,8 @@ fn main() {
             // Uncomment this to override the default log settings:
             // level: bevy::log::Level::TRACE,
             // filter: "wgpu=warn,bevy_ecs=info".to_string(),
+            // This will let you configure file logging
```

Comment:
I think an example showing off some of the settings would be very helpful, rather than just saying to uncomment the line.

Reply:
This will generate artifacts which is not ideal and examples shouldn't have to clean up after themselves.